tsdb/blockwriter_test.go

```diff
@@ -32,6 +32,7 @@ func TestBlockWriter(t *testing.T) {
 	ctx := context.Background()
 	outputDir, err := ioutil.TempDir(os.TempDir(), "output")
 	require.NoError(t, err)
+	defer os.RemoveAll(outputDir)
```

Comment:
Also for the other place
```suggestion
	defer func(){
		require.NoError(t, os.RemoveAll(outputDir))
	}()
```